torch/backends/cudnn/rnn.py

```diff
@@ -252,7 +254,15 @@ def forward(fn, input, hx, weight, output, hy):
         # Alternatively, _copyParams could be written more carefully.
         w.zero_()
         params = get_parameters(fn, handle, w)
+        if fn.skip_input:
+            for layer_index in range(fn.num_directions):
+                params[layer_index][0] = None
+                params[layer_index][2] = None
         _copyParams(weight, params)
+        if fn.skip_input:
+            params = get_parameters(fn, handle, w)
+            for layer_index in range(fn.num_directions):
+                params[layer_index][2].fill_(0)
```

Comment:
Is that necessary? There's `w.zero_()` above, and biases/weights shouldn't be modified by `_copyParams`